### ISSUE TITLE
Not downloadable

### ISSUE THREAD

**Issue by altrisi** [StashCat/Credits#2]
Currently the provided Jenkins installation is down, and the BukkitDev page is empty. Are you going to upload it somewhere (maybe in the Releases tab in Github)? It is a good plugin.

Also: Please upload a version compatible with 1.8 (1.8_R3).
Thanks.

**Comment by StashCat** [comment]
(misclicked the close button)
I'm reinstalling my server, so that's why the jenkins install is down. It's been having issues lately and I'm thinking a clean wipe would be perfect.
For the plugin, the older versions were very basic and, in my opinion, not worth downloading. I'm slowly updating this all, but life issues and business has slowed this all down. However since there is demand for this plugin I'll try to spend a bit more time now. No ETAs though, I have set too many and fulfilled too few.
In other words - hang in tight, all is coming.

**Comment by StashCat** [comment]
I'll see for 1.8 compatible legacy versions, but do note that they probably will not work with any upcoming versions.
